config/express.js

```diff
@@ -70,7 +70,7 @@ module.exports.http = {
       clientID: Reddit.data.clientID,
       clientSecret: Reddit.data.clientIDSecret,
       callbackURL: Reddit.data.redirectURL,
-      scope: "flair,modflair"
+      scope: "flair,modflair,modcontributors,wikiread,wikiedit,read,modposts"
```

Comment:
One option would be to have the app only request the "flair" scope by default, and then each of the mods could manually go to a link [like this](https://www.reddit.com/api/v1/authorize?client_id=PrSdhM2opUJwPg&response_type=code&state=aaaaa&redirect_uri=http://localhost:1337/auth/reddit/callback&duration=permanent&scope=flair,modflair,modcontributors,wikiread,wikiedit,read,modposts) to authenticate the app. That way we wouldn't need users to give any permissions that they didn't need.

Speaking of which, is the "flair" scope even necessary anymore? We could always just have /u/FlairHQ check the user's flair.